functions/z.fish

```diff
@@ -8,7 +8,7 @@ function z -d "jump around"
     Z_STATUS=$?
     echo "$PWD" >&2
     exit $Z_STATUS
-  ' bash $argv ^| read -l Z_PWD
+  ' bash $argv 2>| read -l Z_PWD
```

Comment:
My Bash-fu is weak here, what does `^|` do? This might be a false positive.

Reply:
It pipes stderr to the next command. [See the docs here](http://fishshell.com/docs/current/index.html#piping) … which, incidentally, only show the `2>|` version and not `^|`.